src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -3484,26 +3484,48 @@ public override Ast Copy()
         internal override string GetTooltip()
         {
```

Comment:
Can you cache the tooltip once it's calculated?

Reply:
I've created a `_toolTip` field that stores the string.